industrial_ci/src/tests/clang_format_check.sh

```diff
@@ -38,7 +38,7 @@ function run_clang_format_check() {
     if ! clang-format -style="$CLANG_FORMAT_CHECK" "$file" | git diff --exit-code "$file" - ; then
       err=$[$err +1]
     fi
-  done < <(find "$path" -name '*.h' -or -name '*.hpp' -or -name '*.cpp')
+  done < <(find "$path"/* -name '*.h' -or -name '*.hpp' -or -name '*.cpp')
```

Comment:
Let's add ``-or -name '*.cc'`` here - some code guides (Google) recommend it for C++ files. What do you think @ipa-mdl? Perhaps even ``*.hh``.

Reply:
Good point. Perhaps even `*.c`